pkg/host/internal/sriov/sriov.go

```diff
@@ -437,92 +437,191 @@ func (s *sriov) ConfigSriovDevice(iface *sriovnetworkv1.Interface, ifaceStatus *
 	return nil
 }
 
-func (s *sriov) ConfigSriovInterfaces(storeManager store.ManagerInterface,
-	interfaces []sriovnetworkv1.Interface, ifaceStatuses []sriovnetworkv1.InterfaceExt, pfsToConfig map[string]bool) error {
+func (s *sriov) ConfigSriovInterfaces(storeManager store.ManagerInterface, interfaces []sriovnetworkv1.Interface, ifaceStatuses []sriovnetworkv1.InterfaceExt, pfsToConfig map[string]bool) error {
```

Comment:
discussed with @e0ne offline.

He will check if we can simplify the logic:

1. move check for lockdown here
2. compute interfaces that need configuration and reset
3. configure/rest interfaces either serially or in parallel

that way we will avoid duplicate logic as much as possible.